### ISSUE TITLE
Remove unsed Twitter-Lite implementation

### ISSUE THREAD

**Issue by molebox** [molebox/dev-pack#16]
There is some stale code haging around from when i tried to use twitter-lite, this should be removed

**Comment by liamjosephsilk** [comment]
Happy to pick this up :)

**Comment by molebox** [comment]
> Happy to pick this up :)

Its all yours! If you feel the need to clean up some more on your travels, id suggest noting where you'd like to make changes and creating a separate PR 😊

Let me know if you run into anything 🙌

**Comment by liamjosephsilk** [comment]
I might be missing something as I am just looking at it properly. To my understanding, you are moving away from the twitter cloud function and using the Onegraph login on the front end?

**Comment by molebox** [comment]
> I might be missing something as I am just looking at it properly. To my understanding, you are moving away from the twitter cloud function and using the Onegraph login on the front end?

Yes thats correct. Initially I was going to use the twitter-lite package to update the users profile as we couldnt access it through onegraph. Sean (the guy who created onegraph) helped me out so that it could be done via onegraph instead. This twitter-lite code is now redundant. That being said, there might be reminants of it in the code base so it **might** not be as simple as just removing those files and env vars. In not sure

**Comment by liamjosephsilk** [comment]
I'll take another pass at it and make sure everything is stripped out.

**Comment by molebox** [comment]
> I'll take another pass at it and make sure everything is stripped out.

Hey Liam, id be happy to make you a contributor and give you access to the main repo, that way you can create issues and merge them yourself. Im not really great at managing an open source project and i have alot going on at the same time so if you would like, id be happy to let you take some control on this if you have the time 😊

**Comment by liamjosephsilk** [comment]
> > I'll take another pass at it and make sure everything is stripped out.
> 
> Hey Liam, id be happy to make you a contributor and give you access to the main repo, that way you can create issues and merge them yourself. Im not really great at managing an open source project and i have alot going on at the same time so if you would like, id be happy to let you take some control on this if you have the time 😊

Happy to take a bit of control on this to help out in any way I can :)

**Comment by molebox** [comment]
> > > I'll take another pass at it and make sure everything is stripped out.
> > 
> > 
> > Hey Liam, id be happy to make you a contributor and give you access to the main repo, that way you can create issues and merge them yourself. Im not really great at managing an open source project and i have alot going on at the same time so if you would like, id be happy to let you take some control on this if you have the time 😊
> 
> Happy to take a bit of control on this to help out in any way I can :)

Awesome! Thank you for taking the time and for wanting to get involved. Feel free refactor as you see fit. The code is not very modular and messy in places due to the fact that i never really thought anyone esle would look at it 😬

Looking forward to seeing what you do with it. I'll be around if you have any questions and i will continue to work on it when i have time.
